### PR TITLE
First cut: change args name

### PR DESCRIPTION
**Pull request by TimDaub**
As a reaction to this: https://github.com/bigchaindb/pycoalaip/pull/27

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,6 +42,9 @@
     include_package_data=True,
     install_requires=install_requires,
     tests_require=tests_require,
+    dependency_links=[
+        'git+https://github.com/bigchaindb/pycoalaip.git@make-copyright-holder-after-registration-explicit#egg=pycoalaip-0.0.1.dev2'
```
Comment (TimDaub) [inline setup.py]: Change this to future release of pycoalaip

File: setup.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from setuptools import setup, find_packages
 
 install_requires = [
-    'coalaip>=0.0.1.dev2',
+    'pycoalaip>=0.0.1.dev2',
```
Comment (TimDaub) [inline setup.py]: Change this to future release of pycoalaip

File: setup.py (hunk 1 above)
Comment (TimDaub) [inline setup.py]: Change this to future release of pycoalaip

**Comment by sohkai** [comment]
Probably needs some docs changes too.

**Comment by sohkai** [comment]
Added fixes for tests and documentation. I think we can keep the dependency links in there until we're ready for release.

File: setup.py (hunk 2 above)
Comment (sohkai) [reply setup.py]: Actually the package name is `coalaip`. Will revert.

File: setup.py (hunk 1 above)
Comment (sohkai) [reply setup.py]: We can point to master now, and I'll put this in `dependency_links.txt`.

File: web/views/manifestation.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ def post(self):
         parser = reqparse.RequestParser()
         parser.add_argument('manifestation', type=manifestation_model,
                             required=True, location='json')
-        parser.add_argument('work', type=work_model, required=True,
+        parser.add_argument('copyright_holder', type=work_model, required=True,
```
Comment (sohkai) [inline web/views/manifestation.py]: There's a bit of a conflict with snake case vs camel case here. Should the payload be completely in camel case?

**Comment by sohkai** [comment]
Not sure what's up with travis, but probably something with dependency-link hell since there's a version mismatch between the installed versions of `bigchaindb-driver` and `pycoalaip-bigchaindb`. This should resolve itself once a new version of the driver is released.

File: setup.py (hunk 2 above)
Comment (TimDaub) [reply setup.py]: > Actually the package name is coalaip. Will revert.

doesn't matter for this one, the pip-egg is called pycoalaip, so as long as we use the dependency-links we'll have to go with that.

File: web/views/manifestation.py (hunk 3 above)
Comment (TimDaub) [reply web/views/manifestation.py]: Yes, this should be `CopyrightHolder`

File: setup.py (hunk 2 above)
Comment (sohkai) [reply setup.py]: Mmm, I think the dependency-links don't care about the egg name as long as the name matches what's in setup.py? In any case, `coalaip` works with the current dependency-link.

File: web/views/manifestation.py (hunk 3 above)
Comment (sohkai) [reply web/views/manifestation.py]: Fixed.

**Comment by TimDaub** [comment]
LGTM, do what you think is appropriate regarding the `dependency_links` issue we discussed.

**Comment by sohkai** [comment]
@TimDaub 👍  I'll remove them once we cut new versions of everything.

**Pull request merged**